cron/getIncrementBalance.js

```diff
@@ -0,0 +1,29 @@
+const cron = require('node-cron');
+const { familyModel } = require('../database/models');
+
+const { getLogger } = require('../helpers');
+const logger = getLogger('getIncrementBalance');
+
+async function main() {
+  try {
+    await cron.schedule('1 * * * *', async function () {
+
+      const families = await familyModel.find({});
+
+      families.forEach((item) => {
```

Comment:
This works not as you'd expect )) Results of each `.forEach` callback are not awaited (we're not waiting for promise resolutions here). You should use `.map` here, wrapped into `Promise.all` instead